src/IconButton/IconButton.js

```diff
@@ -200,9 +223,16 @@ class IconButton extends Component {
     const styles = getStyles(this.props, this.context);
     const tooltipPosition = tooltipPositionProp.split('-');
 
+    const hovered = (this.state.hovered || this.state.isKeyboardFocused) && !disabled;
+
+    const mergedRootStyles = Object.assign(
+      styles.root,
+      {backgroundColor: hovered ? hoverColor : transparent},
```

Comment:
Instead of using `transparent` can't we just simply no setting the `backgroundColor`?